### PR TITLE
Refactor ssh key data source and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/data_source_ibm_pi_keys.go [hunk 1]
```diff
@@ -13,78 +13,71 @@ import (
...
 			Attr_Keys: {
-				Type:        schema.TypeList,
 				Computed:    true,
-				Description: "SSH Keys",
+				Description: "List of all the SSH keys.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						Attr_KeyName: {
-							Type:        schema.TypeString,
+						Attr_KeyCreationDate: {
```
Comment (yussufsh) [inline ibm/service/power/data_source_ibm_pi_keys.go:37]: Let's use either of `Attr_KeyCreationDate` or `Attr_CreationDate` in both key and keys data sources.

Comment (ismirlia) [reply ibm/service/power/data_source_ibm_pi_keys.go]: Made changes to Attr_CreationDate

**Review by yussufsh — APPROVED** [review]
lgtm

**Comment by Diptipowervs** [comment]
--- PASS: TestAccIBMPIKeyDataSource_basic (27.23s)
PASS

**Review by hkantare — APPROVED** [review]

**Pull request merged**